app/routes/circulars._archive._index/route.tsx

```diff
@@ -111,23 +112,36 @@ export async function action({ request }: ActionFunctionArgs) {
       if (!createdOnDate || !createdOn)
         throw new Response(null, { status: 400 })
 
-      const {
-        request: { id: zendeskTicketId },
-      } = await postZendeskRequest({
-        requester: { name: user.name, email: user.email },
-        subject: `Change Request for Circular ${circularId}`,
-        comment: {
-          body: `${user.name} has requested an edit. Review at ${origin}/circulars`,
-        },
-      })
+      let zendeskTicketId
+
+      try {
+        zendeskTicketId = (
+          await getChangeRequest(parseFloat(circularId), user.sub)
+        ).zendeskTicketId
+      } catch (error) {
+        console.info('No existing request found')
+      }
```

Comment:
- Catch 404 error (not found) and don't log anything.
- For any other error, just raise it.